### PR TITLE
Modify integration tests for creations and deletions of coordinator tables

### PR DESCRIPTION
**Pull request by brfrn169**
## Description

This PR modifies the integration tests for creations and deletions of coordinator tables.

## Related issues and/or PRs

N/A

## Changes made

- Modified the integration tests to create coordinator tables first
- Modified the integration tests to drop coordinator tables last

## Checklist

- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have updated the documentation to reflect the changes.
- [x] Any remaining open issues linked to this PR are documented and up-to-date (Jira, GitHub, etc.).
- [x] Tests (unit, integration, etc.) have been added for the changes.
- [x] My changes generate no new warnings.
- [x] Any dependent changes in other PRs have been merged and published.

## Additional notes (optional)

N/A

## Release notes

N/A

### CODE REVIEW

File: integration-test/src/main/java/com/scalar/db/api/TwoPhaseCommitTransactionCrossPartitionScanIntegrationTestBase.java [hunk 1]
```diff
@@ -97,9 +97,9 @@ protected String getNamespaceBaseName() {
 
   private void createTables() throws ExecutionException {
     Map<String, String> options = getCreationOptions();
+    admin1.createCoordinatorTables(true, options);
```
Comment (brfrn169) [inline integration-test/src/main/java/com/scalar/db/api/TwoPhaseCommitTransactionCrossPartitionScanIntegrationTestBase.java:100]: Create coordinator tables first.

File: integration-test/src/main/java/com/scalar/db/api/TwoPhaseCommitTransactionCrossPartitionScanIntegrationTestBase.java [hunk 2]
```diff
@@ -125,11 +125,11 @@ public void afterAll() throws Exception {
   }
 
   private void dropTables() throws ExecutionException {
+    admin2.dropTable(namespace2, TABLE_2);
+    admin2.dropNamespace(namespace2);
     admin1.dropTable(namespace1, TABLE_1);
     admin1.dropNamespace(namespace1);
     admin1.dropCoordinatorTables();
```
Comment (brfrn169) [inline integration-test/src/main/java/com/scalar/db/api/TwoPhaseCommitTransactionCrossPartitionScanIntegrationTestBase.java:132]: Drop coordinator tables last.

**Review by komamitsu — APPROVED** [review]
LGTM! 👍

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**